sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/json/JacksonParser.scala

```diff
@@ -286,7 +286,14 @@ class JacksonParser(
           }
 
         case VALUE_NUMBER_INT =>
-          parser.getLongValue * 1000000L
```

Comment:
Can we throw an exception instead?

Reply:
We still need to judge the precise in order to throw exception properly, thus I think it‘s be better to support these two precisions.